### ISSUE TITLE
Repeat mouse till stop

### ISSUE THREAD

**Issue by willwade** [AceCentre/RelayKeys#117]
Want a way of sending a command mouse movement command (say mousemove:10,10) and repeating that until a user sends another command to stopmove

I wasnt sure best way of doing that. It would all have to be threaded.. 

I think we could hack up a little script to call mousemove on repeat and listen for a keyboard shortcut to stop it..  But this feels.. well a hack..

e.g.. (tottally untested)

```
import sys
from pathlib import Path
from pynput import keyboard
import argparse

parser = argparse.ArgumentParser(description='Relaykeys calling script. Repeats a mousemovement in x,y direction till hotkey')
parser.add_argument('--movex', '-x', dest='movex', default=0,
                    help='move in x pixels right')
parser.add_argument('--movey', '-y', dest='movey', default=0,
                    help='move in x pixels down')


def on_activate_stop():
    moveMouse(0,0,False)

def for_canonical(f):
    return lambda k: f(l.canonical(k))

def moveMouse(movex,movey,keepRunning=True):
    while keepRunning:
      # call mousemove:movex,movey
      # Prolly wise to put a little delay in..

hotkey = keyboard.HotKey(
		keyboard.HotKey.parse('<ctrl>+<alt>+s'),
		on_activate_stop)
	with keyboard.Listener(
			on_press=for_canonical(hotkey.press),
			on_release=for_canonical(hotkey.release)) as l:
		l.join()

moveMouse(movex,movey,True)

```

**Comment by joedevsys** [comment]
I can see why this is needed. Terminate with something like any other mouse command (e.g. MouseMove 0,0)  rather than a hotkey might feel less hacky?

**Comment by willwade** [comment]
yeah. Thinking about it I think this would be better dealt with by the RPC daemon as its constantly running. I just need to remember if its blocking or not.

**Comment by willwade** [comment]
So terminating is not that simple - it's easier done by sending a separate app running into the background. I've got a little walkthrough how to do this in the docs. Watch this space.

**Comment by willwade** [comment]
See docs now at https://docs.acecentre.org.uk/products/v/relaykeys/developers/relaykeys-cli#command-mousemove-pixelsright-pixelsdown

NB: when we fix #113 and #115 this will be in the latest release that I'll push
